libs/markets/src/lib/components/market-info/market-info-panels.tsx

```diff
@@ -65,6 +75,7 @@ import {
 } from '@vegaprotocol/network-parameters';
 import type { DataSourceFragment } from './__generated__/MarketInfo';
 import { formatDuration } from 'date-fns';
+import * as AccordionPrimitive from '@radix-ui/react-accordion';
```

Comment:
**question (non-blocking)**: I can see where you need the `AccordionPrimitive.Trigger` down below, but shouldn't it be imported from `@vegaprotocol/ui-toolkit`? The `Accordion` itself already is.

Reply:
I just needed the radix trigger to get the logic for the "ABI spec" expandable area.

Reply:
The ui-toolkit accordion comes with.. accordion styling that I don't need for that small expandable section